docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -46,11 +54,16 @@ Step 1: Prerequisites
 
 * You will need the public keys of all the existing BigchainDB nodes.
 
+* Client Certificate for the new BigchainDB Server to identify itself to MongoDB Primary instance.
+
```

Comment:
BigchainDB Server identifies itself to the cluster and then gets redirected to the MongoDB Primary.